src/models/decisions/decision.ts

```diff
@@ -0,0 +1,17 @@
+import { EnumValue } from "../../helpers/advancedTypes";
+
+export enum VerificationDecision {
+    FALSE = "false",
+    TRUE = "true",
+    UNSURE = "unsure",
+    SKIP = "skip",
+}
+
+// A polymorphic structure that can be used to represent any decision type
+// that implements the interface
+export interface IDecision {
```

Comment:
This interface has overlapping fields with IClassification and IVerification. Why?

Reply:
I have removed these interfaces as per our offline discussion

Using an abstract class was also a good idea as it removes the overlapping fields